db/migrate/20150328144811_consolidate_plant_line_details.rb

```diff
@@ -0,0 +1,195 @@
+class ConsolidatePlantLineDetails < ActiveRecord::Migration
+  def up
+
+    # Rename existing PV to avoid naming clash
+    ActiveRecord::Base.connection.execute("ALTER TABLE plant_varieties \
+      RENAME TO old_plant_varieties")
+
+    create_table :plant_varieties do |t|
+      t.string :plant_variety_name,         primary: true
+      t.string :crop_type
+      t.string :comments
+      t.text :entered_by_whom,              default: "unspecified", null: false
+      t.date :date_entered
+      t.string :data_provenance
+
+      t.string :data_attribution,           default: "unspecified", null: false
+      t.string :country_of_origin,          default: "xxx", null: false
+      t.string :country_registered,         default: "xxx", null: false
```

Comment:
Well. Is countries table used anywhere else in the model? Perhaps we can drop it completely and just leave these two column values as a plain text (like they are now)?